canonical-web-and-design/jaas-dashboard
# Create a component for graphs for controllers overview

## Done
Create a component for graphs for controllers overview

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/controllers
- Check that the charts look like [the design](https://usercontent.irccloud-cdn.com/file/DgXu9v8n/mockup.jpg)
- Check it works responsively
- Check the numbers add up

_The chart is missing the central percentage as the data is below and my change_

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1065

## Screenshots
[if relevant, include a screenshot]
![image](https://user-images.githubusercontent.com/1413534/72482360-57eebb80-37f5-11ea-8885-70f680dd2a58.png)
